.drone.star

```diff
@@ -217,6 +244,10 @@ def jscodestyle():
 
 	return pipelines
 
+	pipelines.append(result)
+
+	return pipelines
+
```

Comment:
The new 4 lines here can be deleted - line 245 already does `return pipelines` and no change is needed to `jscodestyle()`